CHANGELOG.rst

```diff
@@ -11,6 +11,8 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 * Function to apply the quantity discount price (`#707 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/707>`_)
 
+* Function to apply the data management discount price (`#783 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/783>`_)
```

Comment:
This is only about the general DS discount we add with this PR, right? I think the entry above references to the actual function that was added.

```suggestion
* Add data management discount price (`#783 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/783>`_)
```

Reply:
Or something similar